src/PactNet.AspNetCore/PactNet.AspNetCore.csproj

```diff
@@ -6,6 +6,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\PactNet.Abstractions\PactNet.Abstractions.csproj" />
+    <ProjectReference Include="..\PactNet\PactNet.csproj" />
```

Comment:
How come the ASP.Net Core project now depends on the actual implementations project? The aim of the Abstractions project is that plugins/extensions should never need to rely on the actual implementation, and that also allows all the implementation classes to be `internal` instead of `public`

Reply:
That must be a mistake. I went using interfaces to get scenarios and provider states to avoid a tight dependency to PactNet. I'll look into it.